server/views/temporary-accommodation/lost-beds/show.njk

```diff
@@ -0,0 +1,37 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+{% from "moj/components/page-header-actions/macro.njk" import mojPageHeaderActions %}
+{% from "../components/lost-bed-info/macro.njk" import lostBedInfo %}
+{% from "../../partials/breadCrumb.njk" import breadCrumb %}
+{% extends "../../partials/layout.njk" %}
+{% set pageTitle = applicationName + " - Void booking" %}
+{% set mainClasses = "app-container govuk-body" %}
+{% block content %}
+  {{ breadCrumb('View a void booking', [
+    {title: 'List of bedspaces', href: paths.premises.show({ premisesId: premises.id })},
+    {title: 'View bedspace', href: paths.premises.bedspaces.show({ premisesId: premises.id, roomId: room.id })}
+  ]) }}
+  {% include "../../_messages.njk" %}
+  {{ mojPageHeaderActions({
+    heading: {
+      text: 'Void booking',
+      classes: 'govuk-heading-l'
+    },
+    items: actions
+  }) }}
+  <div class="location-header">
+    <h2>Bedspace reference</h2>
+    <p>{{ room.name }}</p>
+    <h2>Property address</h2>
+    <p>{{ premises.addressLine1 }}<br/>{{ premises.postcode }}</p>
+  </div>
```

Comment:
You may well already have this in mind, but let's replace this with the location header component once you've rebased this branch

Reply:
yep 👍